libs/wire-api/src/Wire/API/Error/Galley.hs

```diff
@@ -239,6 +240,8 @@ type instance MapError 'CodeNotFound = 'StaticError 404 "no-conversation-code" "
 
 type instance MapError 'InvalidConversationPassword = 'StaticError 403 "invalid-conversation-password" "Invalid conversation password"
 
+type instance MapError 'ConversationPasswordMissing = 'StaticError 403 "conversation-password-missing" "Conversation password missing"
```

Comment:
this is an api change.  we have a confluence page somewhere where we manually summarize all api changes, have you updated that?